### PR TITLE
Skipping add a comments for SQLite

### PR DESCRIPTION
**Pull request by AnikanovD**
| Q | A |
| --- | --- |
| Fixed issues | #11841 |

### CODE REVIEW

**Comment by SamMousa** [comment]
Nice! Can you add tests?

**Comment by SamMousa** [comment]
Tests should be added here: 
https://github.com/yiisoft/yii2/blob/master/tests/framework/db/ColumnSchemaBuilderTest.php#L25

Ideally you want to add tests for both `comment` and `optionalComment`.
In the test for the sqlite specific builder you could then add a new test for `comment` that checks if an exception is thrown (which is expected behavior).

If you need help regarding tests let me know!

File: framework/db/ColumnSchemaBuilder.php [hunk 1]
```diff
@@ -197,6 +197,17 @@ public function comment($comment)
     }
 
     /**
+     * Specifies the comment for column. If RDBMS does not support comments, then skips.
+     * @param string $comment the comment
+     * @return $this
+     * @since 2.0.
```
Comment (samdark) [inline framework/db/ColumnSchemaBuilder.php]: Note to self: need to fill it before merge.

**Comment by samdark** [comment]
See https://github.com/yiisoft/yii2/commit/01c4dde2026bf106cce2c379bcc5d7f60b79dc5d

**Pull request closed without merging**